sphinx_gallery/interactive_example.py

```diff
@@ -332,6 +334,25 @@ def create_jupyterlite_contents(app, exception):
                         os.path.join(contents_dir, i_folder),
                         ignore=_remove_ipynb_files)
 
+    notebook_modification_function = gallery_conf['jupyterlite'][
+        'notebook_modification_function']
+
+    if notebook_modification_function is None:
+        return
+
+    notebook_pattern = os.path.join(contents_dir, "**", "*.ipynb")
+    notebook_filename_list = glob.glob(notebook_pattern, recursive=True)
+
+    logger.info('Modifying Jupyterlite notebooks ...', color='white')
+    for notebook_filename in notebook_filename_list:
+        with open(notebook_filename) as f:
+            notebook_content = json.load(f)
+
+        notebook_modification_function(notebook_content, notebook_filename)
+
+        with open(notebook_filename, "w") as f:
+            json.dump(notebook_content, f)
```

Comment:
Maybe we should add here `indent=2` as in https://github.com/sphinx-gallery/sphinx-gallery/blob/7f7087d3ee101c0b71b3a92b101db28f3058d667/sphinx_gallery/notebook.py#L347
so that the modified notebook keeps its readability